packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -297,8 +342,11 @@ export class CfnInclude extends core.CfnElement {
         findRefTarget(elementName: string): core.CfnElement | undefined {
           return self.resources[elementName] ?? self.parameters[elementName];
         },
-        findCondition(): undefined {
-          return undefined;
+        findCondition(conditionName): core.CfnCondition | undefined {
+          return self.conditions[conditionName];
+        },
```

Comment:
Whoops! Feels like there's a missing test somewhere if this slipped through the cracks.

Reply:
Nah, it doesn't matter in this particular case, as this will not be called for Outputs. I just didn't like the current declaration. But fair enough, I'll just leave this unchanged in this PR.